src/main/java/com/avail/descriptor/tuples/package-info.java

```diff
@@ -30,6 +30,4 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-@NonnullByDefault
 package com.avail.descriptor.tuples;
```

Comment:
Did you mean to strip out the non-null default for the tuple package?

Reply:
Definitely not. The same thing happened for `com.avail.descriptor.types`. I will fix any other occurrences of this also.